internal/compiler/wasm/wasm.go

```diff
@@ -25,7 +25,7 @@ import (
 const (
 	opaWasmABIVersionVal      = 1
 	opaWasmABIVersionVar      = "opa_wasm_abi_version"
-	opaWasmABIMinorVersionVal = 0
+	opaWasmABIMinorVersionVal = 1
```

Comment:
I guess at some point, we'd want to document what's changed in a minor version bump. 🤔 Perhaps as an addition here? https://www.openpolicyagent.org/docs/edge/wasm/#memory-buffer

Reply:
https://github.com/open-policy-agent/opa/blob/c33e33a41305ec7a28da102d116cb55c7dd021f2/internal/wasm/sdk/opa/capabilities/capabilities.go#L10 This needs updating, too, I suppose. Should perhaps have put a comment here... or we should import that package from here.

Reply:
Done